packages/components/src/templates/next/layouts/Collection/utils/getAvailableFilters.ts

```diff
@@ -0,0 +1,15 @@
+import type { Filter } from "../../../types/Filter"
+import type { ProcessedCollectionCardProps } from "~/interfaces"
+import { getCategoryFilter } from "./getCategoryFilter"
+import { getTagFilters } from "./getTagFilters"
+import { getYearFilter } from "./getYearFilter"
+
+export const getAvailableFilters = (
+  items: ProcessedCollectionCardProps[],
+): Filter[] => {
+  return [
+    getCategoryFilter(items),
+    getYearFilter(items),
+    ...getTagFilters(items),
+  ].filter((filter) => filter.items.length >= 1)
```

Comment:
nit: I think we might be able to remove this filter? There shouldn't be a case where a filter exists but there are 0 items.

Reply:
currently the `get*Filter` functions returned a `Filter` object regardless, and they could be empty `items`

for example, if all the `lastUpdated` are undefined (it's an optional field, though it auto set to Date.now() in Studio UI on initialization) 

thought that we can keep this as a more defensive approach